third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -2840,6 +2879,35 @@ func cidrOrSizeDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 	return strings.HasPrefix(new, "/") && strings.HasSuffix(old, new)
 }
 
+// We want to suppress diffs for the default auto provisioning defaults
```

Comment:
"Defaults" is somewhat overloaded, since https://cloud.google.com/kubernetes-engine/docs/how-to/node-auto-provisioning#identity actually refers to a different set of scopes as defaults. Can we refer to this as suppressing scopes that are automatically added by GKE?

Additionally, I wonder if we can apply the same DSF to `node_config` fields because I believe they have the same behaviour.